src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -508,7 +510,7 @@ public async Task<IActionResult> SaveAnswers(PageViewModel vm, Guid applicationI
                 }
                 else
                 {
-                    return RedirectToAction("TaskList", new { applicationId = applicationId });
+                    return RedirectToAction("TaskList", new { applicationId, sequenceId });
```

Comment:
These changes, along with the TaskList change below look like they'll just cause a double-redirection of the user? Is there not some other way to modify the qs here?

Reply:
every other way involved trying to directly inject the anchor into the url, and if you do this in MVC, in escapes the hashtag character and doesn't work